atlassian/parse.go

```diff
@@ -0,0 +1,109 @@
+package atlassian
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/spyzhov/ajson"
+)
+
+func getTotalSize(node *ajson.Node) (int64, error) {
+	return jsonquery.New(node).ArraySize("issues")
+}
+
+/*
+Records cannot be returned as is from the API. Extra processing is described below.
```

Comment:
What do you mean records cannot be returned as is? Why is that?
Can you add the link to the doc as well?

Reply:
If that is what the API is returning, then we should return as is since that is what the users will be expecting. @laurenzlong Please correct me if I am wrong.

Reply:
I remember some connector long ago had alike problem, hence I went with such approach.